scripts/test_run.sh

```
@@ -148,6 +148,12 @@ resolved_version_from_manifest() {
   echo "$ver"
 }
 
+resolved_version_from_plugin_manifest() {
+  local manifest="$1"
+  local ver="v$(grep '"version"' "$manifest" | sed 's/.*"version"[[:space:]]*:[[:space:]]*"\([^"]*\)".*/\1/')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)